packages/server/graphql/public/types/User.ts

```diff
@@ -174,6 +174,7 @@ const User: UserResolvers = {
         ...activity,
         sortOrder: getScore(activity, teamIds)
       }))
+      .filter((activity) => !type || activity.type === type)
```

Comment:
I could add `type` to the dataloader (or create a second dataloader) so that we only get retrospective templates. But as this query is used in the activity library, which will probably be executed for the org before the user gets to the retro drawer, it should be more efficient to keep it as it is and get the templates from the cache.